src/reset-password/APIFailureMessage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Alert } from '@edx/paragon';
+
+const APIFailureMessage = () => (
+  <div className="d-flex justify-content-center m-4">
+    <div className="d-flex flex-column">
+      <div className="text-center mw-420">
+        <Alert variant="danger">
+          <Alert.Heading className="text-danger">
+            <FormattedMessage
+              id="logistration.reset.password.request.server.error.header.message"
+              defaultMessage="Failed to Reset Password"
+              description="Failed to Reset Password help text heading."
+            />
+          </Alert.Heading>
+          <FormattedMessage
```

Comment:
I guess this will be a common error message for each page, we can move this into common components and pass the heading dynamically for each page.

Also, this message should appear above the form instead of the whole page.